python/provena-summaries.ipynb

```diff
@@ -20,275 +20,353 @@
+      " ItemDataset(display_name='Airflow Example Output Dataset', user_metadata=None, collection_format=CollectionFormat(associations=CollectionFormatAssociations(organisation_id='10378.1/1877551', data_custodian_id=None, point_of_contact=None), approvals=CollectionFormatApprovals(ethics_registration=DatasetEthicsRegistrationCheck(relevant=False, obtained=False), ethics_access=DatasetEthicsAccessCheck(relevant=False, obtained=False), indigenous_knowledge=IndigenousKnowledgeCheck(relevant=False, obtained=False), export_controls=ExportControls(relevant=False, obtained=False)), dataset_info=CollectionFormatDatasetInfo(name='Airflow Example Output Dataset', description='The output dataset from the example airflow workflow', access_info=AccessInfo(reposited=True, uri=None, description=None), publisher_id='10378.1/1877551', created_date=datetime.date(2024, 7, 10), published_date=datetime.date(2024, 7, 10), license=AnyHttpUrl('http://example.com', ), purpose=None, rights_holder=None, usage_limitations=None, preferred_citation=None, spatial_info=None, temporal_info=None, formats=None, keywords=None, user_metadata=None, version=None)), s3=S3Location(bucket_name='restored-dev-dev-rrap-storage-bucket-11102022-11102022', path='datasets/10378-1-1924685/', s3_uri='s3://restored-dev-dev-rrap-storage-bucket-11102022-11102022/datasets/10378-1-1924685/'), release_history=[], release_status=<ReleasedStatus.NOT_RELEASED: 'NOT_RELEASED'>, release_approver=None, release_timestamp=None, access_info_uri=None, history=[HistoryEntry[DatasetDomainInfo](id=0, timestamp=1720586329, reason='Initial record creation', username='ross', item=DatasetDomainInfo(display_name='Airflow Example Output Dataset', collection_format=CollectionFormat(associations=CollectionFormatAssociations(organisation_id='10378.1/1877551', data_custodian_id=None, point_of_contact=None), approvals=CollectionFormatApprovals(ethics_registration=DatasetEthicsRegistrationCheck(relevant=False, obtained=False), ethics_access=DatasetEthicsAccessCheck(relevant=False, obtained=False), indigenous_knowledge=IndigenousKnowledgeCheck(relevant=False, obtained=False), export_controls=ExportControls(relevant=False, obtained=False)), dataset_info=CollectionFormatDatasetInfo(name='Airflow Example Output Dataset', description='The output dataset from the example airflow workflow', access_info=AccessInfo(reposited=True, uri=No
```

Comment:
Nice this makes a really pretty plot and good to have the visual

Reply:
im open to being convinced on keeping cell outputs, particularly the prov graph visualisation in the model-run-client one though if it isnt a privacy concern. I dont think its worth manual effort of clearing/not clearing certain cells though with private info. Better to just clear them all if it is a privacy concern

Reply:
Yeh agreed, I think what we can do is store the image, and then just render that image on the cell afterwards, after clearing out all cell outputs.

Reply:
cool i think i commited the image?